src/main/kotlin/com/arsiu/eduhub/controller/AssignmentController.kt

```diff
@@ -49,7 +48,7 @@ class AssignmentController(
     }
 
     @DeleteMapping("/{id}")
-    fun deleteAssignmentById(@PathVariable id: Long) {
+    fun deleteAssignmentById(@PathVariable id: String) {
```

Comment:
When the function body consists of a single expression, the curly braces can be omitted, as you did in some previous methods. Maybe it's better to keep some one style in classes and don't mix them?)

Reply:
done